connector/src/main/java/org/geysermc/connector/network/translators/chat/MessageTranslator.java

```diff
@@ -158,114 +158,96 @@ public static boolean isMessage(String text) {
     }
 
     /**
-     * Convert a {@link ChatColor} into a string for inserting into messages
+     * Convert a {@link NamedTextColor} into a string for inserting into messages
      *
-     * @param color {@link ChatColor} to convert
+     * @param color {@link NamedTextColor} to convert
      * @return The converted color string
      */
-    private static String getColor(String color) {
-        String base = "\u00a7";
-        switch (color) {
-            case ChatColor.BLACK:
-                base += "0";
-                break;
-            case ChatColor.DARK_BLUE:
-                base += "1";
-                break;
-            case ChatColor.DARK_GREEN:
-                base += "2";
-                break;
-            case ChatColor.DARK_AQUA:
-                base += "3";
-                break;
-            case ChatColor.DARK_RED:
-                base += "4";
-                break;
-            case ChatColor.DARK_PURPLE:
-                base += "5";
-                break;
-            case ChatColor.GOLD:
-                base += "6";
-                break;
-            case ChatColor.GRAY:
-                base += "7";
-                break;
-            case ChatColor.DARK_GRAY:
-                base += "8";
-                break;
-            case ChatColor.BLUE:
-                base += "9";
-                break;
-            case ChatColor.GREEN:
-                base += "a";
-                break;
-            case ChatColor.AQUA:
-                base += "b";
-                break;
-            case ChatColor.RED:
-                base += "c";
-                break;
-            case ChatColor.LIGHT_PURPLE:
-                base += "d";
-                break;
-            case ChatColor.YELLOW:
-                base += "e";
-                break;
-            case ChatColor.WHITE:
-                base += "f";
-                break;
-            case ChatColor.RESET:
-                base += "r";
-                break;
-            default:
-                return "";
+    private static String getColor(NamedTextColor color) {
+        StringBuilder str = new StringBuilder(BASE);
+        if (color.equals(NamedTextColor.BLACK)) {
```

Comment:
Can't you use a `switch` statement here?

Reply:
Unfortunately, because they are values in a class I can't unless I missed something.

Reply:
I would assume you could still do it; you'd just have to type `case NamedTextColor.BLACK`.